test/Interop/Cxx/symbolic-imports/print-symbolic-module-interface.swift

```diff
@@ -98,3 +111,10 @@ public:
 // CHECK-NEXT:     typealias Y = Any
 // CHECK-NEXT:   }
 // CHECK-NEXT: }
+// CHECK: class MyImmortal {
+// CHECK-NEXT:   func foo()
+// CHECK-NEXT: }
+// CHECK-NEXT: struct NonCopyable {
+// CHECK-NEXT:   @available(*, deprecated, message:
+// CHECK-NEXT:   init()
```

Comment:
Wondering why an availability printed for `init()` here but not for other methods?

Reply:
This is the deprecation annotation that we add to the synthesized zero-initializers, when the C++ struct doesn't have a default constructor